crates/apollo-compiler/CHANGELOG.md

```diff
@@ -17,6 +17,93 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## Maintenance
 
 ## Documentation -->
+
+# [0.7.0] (unreleased) - 2023-mm-dd
+
+> Important: X breaking changes below, indicated by **BREAKING**
+
+## BREAKING
+
+### Extensions are “applied” implicitly in HIR, [SimonSapin] in [pull/481], [pull/482], and [pull/484]
```

Comment:
```suggestion
### Extensions are applied implicitly in HIR, [SimonSapin] in [pull/481], [pull/482], and [pull/484]
```
idk it feels nitpicky but i think `applied` is a clear term to use, and the quotes around it made me wonder what we actually mean